src/components/TodoList.jsx

```diff
@@ -1,30 +1,46 @@
 import { Radio } from "./Radio.jsx";
 import { TodoItem } from "./TodoItem.jsx";
-import { useState } from "react";
+import { useState, useRef } from "react";
 export function TodoList() {
-  const [todos, setTodos] = useState([
+  const todoData = [
     {
       id: 1,
       title: "Javascriptの基礎",
       date: "2024-01-01",
       status: "作業中",
-      delete: "✖️",
     },
     {
       id: 2,
       title: "非同期処理",
       date: "2024-01-02",
       status: "作業中",
-      delete: "✖️",
     },
     {
       id: 3,
       title: "オブジェクト指向",
       date: "2024-01-03",
       status: "作業中",
-      delete: "✖️",
     },
-  ]);
+  ];
+  const [todos, settodos] = useState(todoData);
```

Comment:
```suggestion
  const [todos, setTodos] = useState(todoData);
```

CamelCaseにしなきゃいけないです

Reply:
了解です。
修正します！